content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -115,24 +82,85 @@ The Istio control plane must be installed with virtual machine integration enabl
+    {{< /text >}}
+
+1. Get the external ip or internal ip of the VM and store to `"${VM_IP}"`
+
+    {{< text bash >}}
+    $ VM_IP="<the external ip or internal ip of the VM>"
+    {{< /text >}}
+    
+1. Create a new service
+    {{< text bash >}}
+    $ cat <<EOF | kubectl -n $SERVICE_NAMESPACE apply -f -
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: gce-vm
+      labels:
+        app: gce-vm
+    spec:
+      ports:
+      - port: 7070
+        name: grpc
+      - port: 8090
+        name: http
+      selector:
+        app: gce-vm
+    EOF
+    {{< /text >}}
+1. Create workload entry
+
+{{< text bash >}}
+$ cat <<EOF | kubectl -n $namespace apply -f -
+  apiVersion: networking.istio.io/v1beta1
+  kind: WorkloadEntry
+  metadata:
+    name: $VM_NAME
+    namespace: $SERVICE_NAMESPACE
+  spec:
+    address: $VM_IP
+    labels:
+      app: gce-vm
+    serviceAccount: $SERVICE_ACCOUNT
+  EOF
+{{< /text >}}
 
-1. Make a copy of files to copy to the virtual machine
+## Create files to transfer to the virtual machine
 
+1. Create K8s token
+    in the example you can set the  
     {{< text bash >}}
-    $ cp -a "${WORK_DIR}"/"${SERVICE_NAMESPACE}"/key.pem "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/
-    $ cp -a "${WORK_DIR}"/"${SERVICE_NAMESPACE}"/root-cert.pem "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/
-    $ cp -a "${WORK_DIR}"/"${SERVICE_NAMESPACE}"/selfSigned-workload-cert-chain.pem "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/cert-chain.pem
+    $ tokenexpiretime=3600
+    $ echo '{"kind":"TokenRequest","apiVersion":"authentication.k8s.io/v1","spec":{"audiences":["istio-ca"],"expirationSeconds":'$tokenexpiretime'}}' | kubectl create --raw /api/v1/namespaces/$SERVICE_NAMESPACE/serviceaccounts/$SERVICE_ACCOUNT/token -f - | jq -j '.status.token' > "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/istio-token
     {{< /text >}}
 
+1. Get the root cert
+
+{{< text bash >}}
+    $ kubectl -n $SERVICE_NAMESPACE get configmaps istio-ca-root-cert -o json | jq -j '."data"."root-cert.pem"' > "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/root-cert
+{{< /text >}}
+
 1. Generate a `cluster.env` configuration file that informs the virtual machine
    deployment which network CIDR to capture and redirect to the Kubernetes
    cluster:
 
     {{< text bash >}}
-    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ export ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
     $ touch "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/cluster.env
     $ echo ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/cluster.env
```

Comment:
can we just default to `*`? or at least explain this limits capture to only k8s services

Reply:
yes we can set to * it works in both case